code/modules/research/designs/AI_module_designs.dm

```diff
@@ -59,7 +60,7 @@
 	name = "Module Design (Freeform)"
 	desc = "Allows for the construction of a Freeform AI Module."
 	id = "freeform_module"
-	materials = list(/datum/material/glass = 1000, /datum/material/gold = 10000)//Custom inputs should be more expensive to get
+	materials = list(/datum/material/glass = 1000, /datum/material/gold = 10000, /datum/material/bluespace = 2000)//Custom inputs should be more expensive to get
```

Comment:
this should apply for onehuman too so since freeform is mainly used to subvert the AI to yourself.